ruby/chruby/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        postmodern chruby 0.3.9 v
+categories          ruby
+license             MIT
+platforms           darwin
+supported_archs     noarch
+
+maintainers         {catlett.info:chad @chadcatlett} openmaintainer
+
+description         Ruby version changer
+long_description	chruby changes the current Ruby version. It supports bash \
+                    and zsh.
+
+checksums           rmd160  8db66e0e2e29698ee048d1580a337ff60b998f26 \
+                    sha256  5bc542aa03732abec8a6872c9aca003728629f4e2b0ec01062689a4023c6c24c \
+                    size    20925
+
+use_configure       no
+
+build               {}
+
+destroot.cmd        make
+destroot.target     install
```

Comment:
This is unnecessary because it is the default value.

Reply:
removed, sans the `build {}` line.